tests/system/testdata/manifest/apply.txtar

```diff
@@ -16,12 +16,12 @@ output equals ''
 # Successful dry run of a full project.
 ak manifest apply full.yaml --dry-run -J
 return code == 0
-output equals file expected_on_full_dry_run.json
+output equals file expected_on_full_dry_run.txt
```

Comment:
I agree with changing the extension, but can you also remove the unnecessary `-J` from lines 17 and 22?

Reply:
done